xla/service/gpu/transforms/command_buffer_scheduling_test.cc

```diff
@@ -1206,26 +1133,33 @@ TEST_F(CommandBufferSchedulingTest, DynamicSliceFusionStaticSlicing) {
   ENTRY main.9 {
     p0 = s32[2,8,32]{2,1,0} parameter(0)
     p1 = s32[8,32]{1,0} parameter(1)
+    a = s32[128,128] parameter(2)
+    b = s32[128,128] parameter(3)
     c0 = s32[] constant(0)
     c1 = s32[] constant(1)
     slice = s32[1,8,32]{2,1,0} slice(p0), slice={[1:2], [0:8], [0:32]}
     input = s32[8,32]{1,0} reshape(slice)
-    ROOT rs = s32[4,32] reduce-scatter(input), channel_id=64, replica_groups={{0,1}}, use_global_device_ids=true, dimensions={0}, to_apply=add
+    rs = s32[4,32] reduce-scatter(input), channel_id=64, replica_groups={{0,1}}, use_global_device_ids=true, dimensions={0}, to_apply=add
+    dot = s32[128,128] dot(a,b), lhs_contracting_dims={1}, rhs_contracting_dims={0}
+    ROOT tuple = tuple(rs, dot)
   })";
 
-  TF_ASSERT_OK_AND_ASSIGN(auto m, GetOptimizedModule(hlo));
+  TF_ASSERT_OK_AND_ASSIGN(std::unique_ptr<HloModule> m,
+                          ParseAndReturnVerifiedModule(hlo));
```

Comment:
nit: maybe we could create the config before calling `ParseAndReturnVerifiedModule` and pass it as an additional parameter?